halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/security/authn/saml/EditSamlCommand.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.cli.command.v1.config.security.authn.saml;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.netflix.spinnaker.halyard.cli.command.v1.config.security.authn.AbstractEditAuthnMethodCommand;
+import com.netflix.spinnaker.halyard.config.error.v1.IllegalConfigException;
+import com.netflix.spinnaker.halyard.config.model.v1.security.AuthnMethod;
+import com.netflix.spinnaker.halyard.config.model.v1.security.Saml;
+import lombok.Getter;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+@Parameters(separators = "=")
+public class EditSamlCommand extends AbstractEditAuthnMethodCommand<Saml> {
+
+  @Getter
+  private AuthnMethod.Method method = AuthnMethod.Method.SAML;
+
+  @Parameter(
+      names = "--metadata",
+      description = "The address to your identity provider's metadata XML file. This can be a URL" +
```

Comment:
This needs to be prefixed with `http` or `file`, right? If so, can you call that out here?